Formula/trino.rb

```diff
@@ -21,25 +21,23 @@ class Trino < Formula
   depends_on "python@3.11"
 
   resource "trino-src" do
-    url "https://github.com/trinodb/trino/archive/refs/tags/406.tar.gz", using: :nounzip
-    sha256 "647d08c00693113f41fe600718b3436ee6c6738046876ea14796ed84230a0fd2"
+    url "https://github.com/trinodb/trino/archive/refs/tags/407.tar.gz", using: :nounzip
+    sha256 "e8a777efcab913ed80bd2a08ad18a5afaf9338e0ab15fc996edaa74f5863e11b"
   end
 
   resource "trino-cli" do
-    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/406/trino-cli-406-executable.jar"
-    sha256 "70f7ab57cf74f655942d14f4baafebac32ca64ca833790411691a69b574c16bb"
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/407/trino-cli-407-executable.jar"
+    sha256 "1b1fd386501758b7a392a4791e49da061614b2890d3bf76ecfa130aa56380912"
   end
 
   def install
     # Manually extract tarball to avoid losing hardlinks which increases bottle
     # size from MBs to GBs. Remove once Homebrew is able to preserve hardlinks.
-    # Ref: https://github.com/Homebrew/brew/pull/13154
     libexec.mkpath
     system "tar", "-C", libexec.to_s, "--strip-components", "1", "-xzf", "trino-server-#{version}.tar.gz"
 
     # Manually untar, since macOS-bundled tar produces the error:
     #   trino-363/plugin/trino-hive/src/test/resources/<truncated>.snappy.orc.crc: Failed to restore metadata
-    # Remove when https://github.com/trinodb/trino/issues/8877 is fixed
     resource("trino-src").stage do |r|
```

Comment:
If upstream thinks it's fixed, why do we still do this?

Reply:
the issue is actually fixed, but got closed. https://github.com/trinodb/trino/issues/8877 (I guess we just did not do the followup)

Reply:
Actually this does work for me now, let me update the PR

```
$ curl -sSL https://github.com/trinodb/trino/archive/360.tar.gz | tar -xz
$
```

Reply:
actually still an issue. The zip file does not work either